### ISSUE TITLE
`PylancereportGeneralTypeIssues` when attempting to use with an `AsyncExitStack`

### ISSUE THREAD

**Issue by sdwilsh** [sbtinstruments/asyncio-mqtt#98]
When attempting to use an `AsyncExitStack`, the typing of `Client` is not compatible, and causes pylance to report an error:
```
Argument of type "Client" cannot be assigned to parameter "cm" of type "AbstractAsyncContextManager" in function "enter_async_context"
  "Client" is incompatible with protocol "AbstractAsyncContextManager"
    "__aexit__" is an incompatible type
      Type "(exc_type: Type[Exception], exc: Exception, tb: TracebackType) -> Coroutine[Any, Any, None]" cannot be assigned to type "(__exc_type: Type[BaseException] | None, __exc_value: BaseException | None, __traceback: TracebackType | None) -> Awaitable[bool | None]"
        Parameter 1: type "Type[BaseException] | None" cannot be assigned to type "Type[Exception]"
          Type "Type[BaseException] | None" cannot be assigned to type "Type[Exception]"
        Parameter 2: type "BaseException | None" cannot be assigned to type "Exception"
          Type "BaseException | None" cannot be assigned to type "Exception"
        Parameter 3: type "TracebackType | None" cannot be assigned to type "TracebackType"
```

**Comment by sdwilsh** [comment]
This was fixed in 526ede4a7b3a2d267c061bde383eda4c4f5993b3 by @laundmo. We just need a new release :)

**Comment by frederikaalund** [comment]
I just released v0.11.1. Better late than never. :)
